README_cucumber.md

```diff
@@ -1,7 +1,6 @@
-# Department for Education
-
-## Early Years Foundation Stage (EYFS) Test Automation (Functional)
```

Comment:
Does this change really add value?

Reply:
We know what the project is called !

Reply:
People stop reading bloated READMEs, they need to be terse

Reply:
1. I've put the URL back in support/env.rb and renamed it TARGET_URL.  Users do not need to log in.
2. I can't find the words 'is it Friday' in the project, its not in this PR as far as I know
3. I've delete landing_page.feature, good spot. I was going to remove the eyfs_ prefix from the feature file names
4. See 1
5. The user does not need to log in for these tests to work, so now it just visits the TARGET_URL 
6. & 7.  I must have deleted and then re-added these files. 
8. Commented out code either needs to either be made to work or removed.  It will still be in the git history for future reference
9. The "EYFS Service" user was removed because it is not a user and does not login. The step just visits a page
11. StandardError will catch any exception
12. I went a round about way to include this code and somehow ended up removing and repeating some of the rubocop fixes.